README.md

```diff
@@ -3,14 +3,11 @@
 ![Go Coverage](https://github.com/Layr-Labs/eigensdk-go/wiki/coverage.svg)
 [![Go Report Card](https://goreportcard.com/badge/github.com/Layr-Labs/eigensdk-go)](https://goreportcard.com/report/github.com/Layr-Labs/eigensdk-go)
 
-<p align="center"><b>
-🚧 Under active development. EIGENSDK-GO is rapidly being upgraded, features are being added, interfaces will have breaking changes 🚧
-</b><p>
 
-**Do not use it in Production, testnet only.**
+
 
 ## EigenSDK
-This SDK provides a set of primitive Go modules for developing AVSs used in EigenLayer
+This SDK provides a set of primitive Go modules for developing AVSs used in EigenLayer.
```

Comment:
« in Eigenlayer » sounds a bit weird to me. Is there another formulation we could use instead?

Reply:
Agreed, changed to "on" vs "used in"